otns_main/otns_main.go

```diff
@@ -143,13 +143,14 @@ func Main(visualizerCreator func(ctx *progctx.ProgCtx, args *MainArgs) visualize
 	}
 
 	visGrpcServerAddr := fmt.Sprintf("%s:%d", args.DispatcherHost, args.DispatcherPort-1)
+	replayFn := fmt.Sprintf("otns_%s.replay", os.Getenv("PORT_OFFSET"))
 	if vis != nil {
 		vis = visualizeMulti.NewMultiVisualizer(
 			vis,
-			visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr),
+			visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, "replayFn"),
 		)
 	} else {
-		vis = visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr)
+		vis = visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, replayFn)
```

Comment:
What is the difference between
```go
visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, "replayFn")
```
and
```go
visualizeGrpc.NewGrpcVisualizer(visGrpcServerAddr, replayFn)
```
?

Reply:
It's good interview question.

Reply:
fixed